### PR TITLE
Add support for breadcrumbs structured data #2

### PR DESCRIPTION
**Pull request by tgerulaitis**
screenshot 2016-07-19 16 58 07

### CODE REVIEW

File: src/app/design/frontend/base/default/layout/meanbee/osd.xml [hunk 1]
```diff
@@ -1,5 +1,11 @@
 <?xml version="1.0"?>
 <layout>
+    <default>
+        <reference name="head">
+            <block type="meanbee_osd/breadcrumbs" name="meanbee.osd.breadcrumbs" ifconfig="meanbee_osd/general/breadcrumbs_enabled" />
```
Comment (bobbyshaw) [inline src/app/design/frontend/base/default/layout/meanbee/osd.xml]: I think ifconfig only works on actions not blocks?

File: src/app/code/community/Meanbee/OSD/Helper/Config.php [hunk 2]
```diff
@@ -36,6 +37,18 @@ public function isEnabled($store = null)
...
     /**
+     * Check if breadcrumbs are enabled in the organisational data output.
+     *
+     * @param Mage_Core_Model_Store|int|null $store
+     *
+     * @return bool
+     */
+    public function isBreadcrumbsEnabled($store = null)
+    {
+        return Mage::getStoreConfigFlag(static::XML_PATH_MODULE_BREADCRUMBS_ENABLED, $store);
```
Comment (bobbyshaw) [inline src/app/code/community/Meanbee/OSD/Helper/Config.php:48]: Should the current enabled flag be a master switch for the entire module?  At the moment, the module is defaulted to off but breadcrumbs to enabled.

**Comment by bobbyshaw** [comment]
Couple of thoughts but approved!

File: src/app/code/community/Meanbee/OSD/Helper/Config.php (hunk 2 above)
Comment (tgerulaitis) [inline src/app/code/community/Meanbee/OSD/Helper/Config.php:48]: Good point!

File: src/app/design/frontend/base/default/layout/meanbee/osd.xml (hunk 1 above)
Comment (tgerulaitis) [inline src/app/design/frontend/base/default/layout/meanbee/osd.xml]: Well that is stupid of Magento, isn't it?...

**Comment by tgerulaitis** [comment]
Force-pushed an update to keep the history clean.

**Pull request merged**